src/theme.ts

```diff
@@ -12,19 +12,22 @@ const palette = {
     // green colors
     main: "#006778",
     light: "#c2dbdf",
+    lighter: "#c2dbdf",
     lightest: "#e5eff1",
   },
   secondary: {
     // grey colors
     main: "#595959",
     light: "#dfe1e3",
+    lightest: "#DFE1E3",
   },
   background: {
-    default: "#e5eff1",
+    default: "#d8e8ea",
```

Comment:
According to the design, I think we have at least for Login and Home page - the main background color is: `#e5eff1`. For Wizard pages, main background color is: `#d8e8ea`. We have different background colors for different cases, maybe it's best to ask @ainoc for clarification.

Reply:
Good catch. I extended another background color value as `wizard` so we're able to have different background colors for different cases. @ainoc, do you have any suggestions or ideas if this is something that we'd want?

Reply:
Have you got the answer for this or I got this confirmation from @ainoc

![Screenshot 2022-05-06 at 9 07 31](https://user-images.githubusercontent.com/23532204/167076344-ece5156c-cdd9-4a0b-bdd5-1d8af101252c.png)
: